packages/ui/src/components/Form/styles.css.ts

```diff
@@ -1,6 +1,7 @@
 import { recipe, RecipeVariants } from '@vanilla-extract/recipes'
 import { colors } from '../../css/colors'
 import { vars } from '../../css/vars'
+import { typographyStyles } from '@components/Text/styles.css'
```

Comment:
looks like this is causing the [js build to fail](https://github.com/highlight/highlight/actions/runs/4772287183/jobs/8484704824?pr=5030)

```
Error: @highlight-run/frontend:typegen: ../packages/ui/src/components/Form/styles.css.ts(4,10): error TS2614: Module '"*.css"' has no exported member 'typographyStyles'. Did you mean to use 'import typographyStyles from "*.css"' instead?
```

i think the import needs to change in this case to
`import typographyStyles from "*.css"`

Reply:
Thank you @Vadman97, i've resolved the issue.